ActiveRecord.php

```diff
@@ -415,7 +415,7 @@ public static function instantiate($row)
      * By default the `op_type` is set to `create`.
      * @return boolean whether the attributes are valid and the record is inserted successfully.
      */
-    public function insert($runValidation = true, $attributes = null, $options = ['op_type' => 'create'])
+    public function insert($runValidation = true, $attributes = null)
```

Comment:
Seems there isn't. Need to return `$options` parameter to support parent/routing options for indexing API.